packages/contracts/contracts/ChugSplashRegistry.sol

```diff
@@ -78,12 +78,12 @@ contract ChugSplashRegistry is Ownable, Initializable {
     /**
      * @notice Mapping of claimers to project names to ChugSplashManagerProxy contracts.
      */
-    mapping(address => mapping(bytes32 => ChugSplashManagerProxy)) public projects;
+    mapping(address => mapping(bytes32 => address payable)) public projects;
```

Comment:
I changed this because it makes testing slightly simpler. Specifically, in `ChugSplash.t.sol`, we can write this:
```
ChugSplashManager manager = ChugSplashManager(registry.projects(claimer, transferOrganizationID));
```

Instead of this:
```
ChugSplashManager manager = ChugSplashManager(payable(address(registry.projects(claimer, transferOrganizationID))));
```